File: exercise/examples/perfect.c
#include <stdio.h>
int main (void)
{
	int result = 0 ;
	int number ;
	scanf ("%d",&number);
	int i = 1;
	while (i<number)
	{
		if (number % i == 0)
		{
			result +=i;
		}
		i++;
	}
	if (result == number)
	{
		printf ("%d is a perfect number...\n",number);
	}
	else 
	{
		printf ("%d is not a perfect number...\n",number);
	}
	return 0;

}

File: exercise/examples/perfect.py
number = int (raw_input ("Enter the number :"))
summation = 0
for i in range (1,number):
    if (number%i==0):
        summation += i
if (summation==number):
    print ("{} is perfect number...".format(number))
else :
    print ("{} is not a perfect number...".format(number))
        
